### PR TITLE
Move older releases to separate section

### PR DESCRIPTION
**Pull request by TimvdLippe**
To prevent any confusion and make it clearer 2.x is now the version developed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ Tasty mocking framework for unit tests in Java
...
-
-![latest 2.x](https://img.shields.io/badge/latest beta-2.x =>-yellow.svg) [ ![Current release](https://api.bintray.com/packages/mockito/maven/mockito/images/download.svg) ](https://bintray.com/mockito/maven/mockito/_latestVersion) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core/badge.svg?style=flat)](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core)
+![latest 2.x](https://img.shields.io/badge/latest 2.x =>-yellow.svg) [ ![Current release](https://api.bintray.com/packages/mockito/maven/mockito/images/download.svg) ](https://bintray.com/mockito/maven/mockito/_latestVersion) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core/badge.svg?style=flat)](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core)
```
Comment (bric3) [inline README.md]: This link is wrong see the rich diff.

Replace by https://img.shields.io/badge/latest-2.x%20=%3E-yellow.svg

Comment (TimvdLippe) [inline README.md]: Oops should be fixed

File: README.md [hunk 2]
```diff
@@ -8,13 +8,14 @@ Tasty mocking framework for unit tests in Java
...
-
-![latest 2.x](https://img.shields.io/badge/latest beta-2.x =>-yellow.svg) [ ![Current release](https://api.bintray.com/packages/mockito/maven/mockito/images/download.svg) ](https://bintray.com/mockito/maven/mockito/_latestVersion) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core/badge.svg?style=flat)](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core)
+![latest 2.x](https://img.shields.io/badge/latest-2.x =>-yellow.svg) [ ![Current release](https://api.bintray.com/packages/mockito/maven/mockito/images/download.svg) ](https://bintray.com/mockito/maven/mockito/_latestVersion) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core/badge.svg?style=flat)](https://maven-badges.herokuapp.com/maven-central/org.mockito/mockito-core)
```
Comment (szpak) [inline README.md]: Maybe we could point directly 2.1.0 in Maven Central (to no confuse people with 2.2.0-beta.1)?

As a drawback we would need to switch it manually when 2.2.0 is released.

Comment (bric3) [inline README.md]: That's not a bad idea !

Comment (TimvdLippe) [inline README.md]: On it!

Comment (TimvdLippe) [inline README.md]: The diff is a bit weird, but it should be just fine :smile:

**Comment by szpak** [comment]
Please remember also about the `2.x` branch.

**Comment by bric3** [comment]
I think that our build script can perform the readme update at some point.

**Comment by bric3** [comment]
LGTM

**Comment by szpak** [comment]
@TimvdLippe Are you still working on this or it can be merged?

**Comment by TimvdLippe** [comment]
I actually someone else was merging since this is my own PR. I generally do not merge my own PRs. Shall I anyways?

**Comment by bric3** [comment]
Go on we approved your changes.

**Pull request merged**